drivers/watchdog/wdt_tco.c

```diff
@@ -267,9 +267,9 @@ static struct tco_data wdt_data = {
 };
 
 static const struct tco_config wdt_config = {
-	.base = DT_REG_ADDR(TCO_WDT_NODE),
+	.base = DT_INST_REG_ADDR(0),
```

Comment:
I was thinking more in terms of just a one line change: `#define TCO_WDT_NODE DT_INST(0)` - wouldn't that work?

Reply:
I check current code and have not found this pattern.

Reply:
Hardcoding the "magic" `0` number in multiple places doesn't seem ideal to me, whereas having the clear define (like the driver has now) seems like the most intuitive way of communicating to the reader that only a single instance is supported.

Reply:
@finikorg can you point me to some of these "single instance supported" drivers that scatter the `0` value across the driver code?

Reply:
I just searched for "DT_INST_DEFINE(0". However I can use proposed approach with DT_INST(0), I just do not see any pattern of using it.